rep-0152.rst

```diff
@@ -0,0 +1,144 @@
+---------------------------------------
+
+If you can no longer maintain a your package.
+
+a. Set the package maintainer section in the ``package.xml`` file as follows.
+
+``<maintainer email="ros-orphaned-packages@googlegroups.com">ROS Orphaned Package Maintainers</maintainer>``
+
+b. Send email to ros-orphaned-packages@googlegroups.com and/or post to the `ROS Discourse Release Category <http://discourse.ros.org/c/release>`_, indicating that package is now orphaned.
+Wait response from someone in the ROS Orphaned Package Maintainer team.
+
+c. Add admin permission of both source and release repository to who respond from in the ROS Orphaned Package Maintainers team.
+
+1-B. Orphaning a package by users
+---------------------------------
+
+If you find a package is not actively maintained and willing to release as a ROS Orphaned Package Maintainer.
+
+a. Contact the current maintainer, ask for transition to orphaned package and request access to source and release repository in order to make releases.
+
+b. If the current maintainer is not responsible (a single day isn't enough to conclude that) and you still want to release the packages, there are two different paths forward depend on the "location" of the source and release repositories.
+
+   1. If the repositories are hosted on GitHub within an `ros-` org unit, the admins of the org unit will usually grant the necessary access to these repositories.
+
+   2. If nobody else beside the original author (which doesn't respond) has access, you should fork the repository.
+
+c. Add ROS Orphaned Package Maintainers to the maintainer section of package.xml as follows.
+
+
+``<maintainer email="ros-orphaned-packages@googlegroups.com">ROS Orphaned Package Maintainers</maintainer>``
+
+2. Releasing orphaned packages
+------------------------------
+
+a. The package release process is as same as normal packages, so please follow bloom tutorial titled ``Releasing a Package for the First Time`` in ROS wiki documents [9]_ to release the orphaned package.
+
+
+   1. Updated maintainer section in ``package.xml`` indicating that the package is orphaned and commit to the source tree, before you run the ``catkin_generate_changelog`` command.
+
+   2. During ``bloom-release`` process, you will asked to set maintenance status as follows. Choose ``unmaintained`` . ::
+      
+::
+
+  Would you like to add a maintenance status for this repository?' [Y/n] Y
+   Please enter a maintenance status.
+   Valid maintenance statuses:
+   - developed: active development is in progress
+   - maintained: no new development, but bug fixes and pull requests are addressed
+   - unmaintained: looking for new maintainer, bug fixes and pull requests will not be addressed
+   - end-of-life: should not be used, will disappear at some point
+
+b. Add your package to the list of orphaned packages in `ROS OrphanedPackage page <http://wiki.ros.org/OrphanedPackage>`_ of ROSwiki.
+
+3. Adopting packages
```

Comment:
I think this should be mentioned before `2. Releasing orphaned packages` since it is clearly the preferred solution.